kas-installer-defaults.env

```diff
@@ -6,12 +6,12 @@ trap 'echo "** Unbound variable must be configured in kas-installer.env or user
 RH_USERNAME="${RH_USERNAME}"
 
 # [required]
-# Owning users RH account_id. See `str=$(ocm token | awk -F'.' '{print $2}')===; echo ${str:0:${#str}&-4} | base64 -d | jq -r .account_id`
+# Owning users RH account_id. See `str=$(ocm token --payload | jq -r .account_id)`
```

Comment:
I think you can even go a step further and remove the `str=$(` and `)`. The `str` variable was only present to support the bash substring that isn't present anymore.